src/cosmicqc/scdataframe.py

```diff
@@ -128,6 +136,280 @@ def export(
         else:
             raise ValueError("Unsupported file format for export.")
 
+    @staticmethod
+    def is_notebook_or_lab() -> bool:
+        """
+        Determines if the code is being executed in a Jupyter notebook or
+        JupyterLab environment.
```

Comment:
This comment was a bit confusing to me. I read this as there could only be two options (notebook or lab). However, reading below I see that it's testing if in jupyter or by some other mechanism (e.g., CLI). Consider making this docstring a bit more clear.

Reply:
Thanks, will do, I'll make this more clear.